toolchain/sem_ir/stringify.cpp

```diff
@@ -325,8 +326,15 @@ class Stringifier {
         sem_ir_->facet_types().Get(inst.facet_type_id);
     // Output `where` restrictions.
     bool some_where = false;
-    if (facet_type_info.other_requirements) {
-      step_stack_->PushString("...");
+    if (facet_type_info.special_requirements_mask.has(
+            SemIR::FacetTypeInfo::SpecialRequirement::Other)) {
+      step_stack_->PushString(" ...");
+      some_where = true;
+    }
+    if (facet_type_info.special_requirements_mask.has(
+            SemIR::FacetTypeInfo::SpecialRequirement::
+                TypeCanAggregateDestroy)) {
+      step_stack_->PushString(" CanAggregateDestroy");
```

Comment:
Again, it looks to me like this should take the place of the interface, not a where subexpression?